src/pretalx/orga/templates/orga/event/dashboard.html

```diff
@@ -16,7 +16,7 @@
                     <li><a href="{{ request.event.orga_urls.team_settings }}">
                             {% trans "Gather your team" %}
                     </a></li>
-                    <li><a href="{{ request.event.orga_urls.cfp }}">
+                    <li><a href="{{ request.event.orga_urls.edit_cfp }}">
```

Comment:
So this URL should be `request.event.cfp.urls.text` instead.

Reply:
yeah I figured it would work that way. Just thought it odd to have all "orga_urls." except for the cfp. But yeah..